diff --git a/CMakeLists.txt b/CMakeLists.txt
index 92c50c2e2..c2652de7b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -204,7 +204,7 @@ elseif(APPLE)
       # for Mac OS X, everything is put inside an application bundle
       SET (CMAKE_INSTALL_PREFIX ${CMAKE_INSTALL_PREFIX}/)
       # path for library references
-      SET (CMAKE_INSTALL_NAME_DIR @executable_path/lib)
+      SET (CMAKE_INSTALL_NAME_DIR ${CMAKE_INSTALL_PREFIX}/lib)
       # install the Info.plist file
       install(FILES src/mac/Info.plist DESTINATION ${CMAKE_INSTALL_PREFIX}/Marble.app/Contents)
       #SET (lib_dir ${CMAKE_INSTALL_PREFIX}/Marble.app/Contents/MacOS/lib)
diff --git a/src/lib/astro/CMakeLists.txt b/src/lib/astro/CMakeLists.txt
index 27c4e4f62..ead8ccda7 100644
--- a/src/lib/astro/CMakeLists.txt
+++ b/src/lib/astro/CMakeLists.txt
@@ -73,7 +73,7 @@ endif()
 
 if (APPLE)
     # TODO: find how to install & deploy library, headers and config files as bundle for development
-    return()
+#    return()
 endif()
 
 
@@ -92,24 +92,24 @@ endif()
 
 ########### CMake Config files ###############
 
-include(CMakePackageConfigHelpers)
-
-set(ASTRO_INSTALL_CMAKECONFIGDIR ${CMAKE_INSTALL_LIBDIR}/cmake/${ASTRO_CMAKECONFIGNAME})
-
-configure_package_config_file(AstroConfig.cmake.in
-    ${CMAKE_CURRENT_BINARY_DIR}/${ASTRO_CMAKECONFIGNAME}Config.cmake
-    INSTALL_DESTINATION ${ASTRO_INSTALL_CMAKECONFIGDIR}
-)
-
-install(FILES
-    ${CMAKE_CURRENT_BINARY_DIR}/${ASTRO_CMAKECONFIGNAME}Config.cmake
-    ${CMAKE_CURRENT_BINARY_DIR}/${ASTRO_CMAKECONFIGNAME}ConfigVersion.cmake
-
-    DESTINATION ${ASTRO_INSTALL_CMAKECONFIGDIR}
-)
-
-install(
-    EXPORT AstroTargets
-    FILE ${ASTRO_CMAKECONFIGNAME}Targets.cmake
-    DESTINATION ${ASTRO_INSTALL_CMAKECONFIGDIR}
-)
+#include(CMakePackageConfigHelpers)
+#
+#set(ASTRO_INSTALL_CMAKECONFIGDIR ${CMAKE_INSTALL_LIBDIR}/cmake/${ASTRO_CMAKECONFIGNAME})
+#
+#configure_package_config_file(AstroConfig.cmake.in
+#    ${CMAKE_CURRENT_BINARY_DIR}/${ASTRO_CMAKECONFIGNAME}Config.cmake
+#    INSTALL_DESTINATION ${ASTRO_INSTALL_CMAKECONFIGDIR}
+#)
+#
+#install(FILES
+#    ${CMAKE_CURRENT_BINARY_DIR}/${ASTRO_CMAKECONFIGNAME}Config.cmake
+#    ${CMAKE_CURRENT_BINARY_DIR}/${ASTRO_CMAKECONFIGNAME}ConfigVersion.cmake
+#
+#    DESTINATION ${ASTRO_INSTALL_CMAKECONFIGDIR}
+#)
+#
+#install(
+#    EXPORT AstroTargets
+#    FILE ${ASTRO_CMAKECONFIGNAME}Targets.cmake
+#    DESTINATION ${ASTRO_INSTALL_CMAKECONFIGDIR}
+#)
diff --git a/src/lib/marble/CMakeLists.txt b/src/lib/marble/CMakeLists.txt
index 51691b61c..454b46ed5 100644
--- a/src/lib/marble/CMakeLists.txt
+++ b/src/lib/marble/CMakeLists.txt
@@ -469,7 +469,7 @@ endif()
 
 if (APPLE)
     # TODO: find how to install & deploy library, headers and config files as bundle for development
-    return()
+#    return()
 endif()
 
 if (MARBLE_NO_WEBKITWIDGETS)
@@ -649,27 +649,27 @@ install( FILES
 
 ########### CMake Config files ###############
 
-include(CMakePackageConfigHelpers)
-
-set(MARBLE_INSTALL_CMAKECONFIGDIR ${CMAKE_INSTALL_LIBDIR}/cmake/${MARBLE_CMAKECONFIGNAME})
-
-configure_package_config_file(MarbleConfig.cmake.in
-    ${CMAKE_CURRENT_BINARY_DIR}/${MARBLE_CMAKECONFIGNAME}Config.cmake
-    INSTALL_DESTINATION ${MARBLE_INSTALL_CMAKECONFIGDIR}
-)
-
-install(FILES
-    ${CMAKE_CURRENT_BINARY_DIR}/${MARBLE_CMAKECONFIGNAME}Config.cmake
-    ${CMAKE_CURRENT_BINARY_DIR}/${MARBLE_CMAKECONFIGNAME}ConfigVersion.cmake
-
-    DESTINATION ${MARBLE_INSTALL_CMAKECONFIGDIR}
-)
-
-install(
-    EXPORT MarbleTargets
-    FILE ${MARBLE_CMAKECONFIGNAME}Targets.cmake
-    DESTINATION ${MARBLE_INSTALL_CMAKECONFIGDIR}
-)
+#include(CMakePackageConfigHelpers)
+#
+#set(MARBLE_INSTALL_CMAKECONFIGDIR ${CMAKE_INSTALL_LIBDIR}/cmake/${MARBLE_CMAKECONFIGNAME})
+#
+#configure_package_config_file(MarbleConfig.cmake.in
+#    ${CMAKE_CURRENT_BINARY_DIR}/${MARBLE_CMAKECONFIGNAME}Config.cmake
+#    INSTALL_DESTINATION ${MARBLE_INSTALL_CMAKECONFIGDIR}
+#)
+#
+#install(FILES
+#    ${CMAKE_CURRENT_BINARY_DIR}/${MARBLE_CMAKECONFIGNAME}Config.cmake
+#    ${CMAKE_CURRENT_BINARY_DIR}/${MARBLE_CMAKECONFIGNAME}ConfigVersion.cmake
+#
+#    DESTINATION ${MARBLE_INSTALL_CMAKECONFIGDIR}
+#)
+#
+#install(
+#    EXPORT MarbleTargets
+#    FILE ${MARBLE_CMAKECONFIGNAME}Targets.cmake
+#    DESTINATION ${MARBLE_INSTALL_CMAKECONFIGDIR}
+#)
 
 ########### Qt Pri file ###############
 
