diff --git a/src/corelib/CMakeLists.txt b/src/corelib/CMakeLists.txt
index 403654d0fc..996e53ee09 100644
--- a/qtbase/src/corelib/CMakeLists.txt
+++ b/qtbase/src/corelib/CMakeLists.txt
@@ -758,7 +758,9 @@ qt_internal_extend_target(Core CONDITION QT_FEATURE_icu
         text/qcollator_icu.cpp
         text/qlocale_icu.cpp
     LIBRARIES
-        ICU::i18n ICU::uc ICU::data
+        /usr/local/lib/libicui18n.so
+        /usr/local/lib/libicuuc.so
+        /usr/local/lib/libicudata.so
 )
 
 qt_internal_extend_target(Core CONDITION WIN32 AND NOT QT_FEATURE_icu
