diff --git a/CMakeLists.txt b/CMakeLists.txt
index b50b739..f34f0d8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -148,8 +148,8 @@ include_directories (${CMAKE_CURRENT_SOURCE_DIR}/include)
 install (FILES include/exchndl.h DESTINATION include)
 
 add_subdirectory (src)
-add_subdirectory (sample)
-add_subdirectory (tests)
+#add_subdirectory (sample)
+#add_subdirectory (tests)
 
 
 ##############################################################################
diff --git a/cmake/StaticCRT.cmake b/cmake/StaticCRT.cmake
index 1e9c90a..6cb5b06 100644
--- a/cmake/StaticCRT.cmake
+++ b/cmake/StaticCRT.cmake
@@ -19,17 +19,17 @@ if (MINGW)
         endif ()
 
         # Statically link Posix threads when detected.
-        execute_process (
-            COMMAND "${CMAKE_COMMAND}" -E echo "#include <thread>\n#ifdef _GLIBCXX_HAS_GTHREADS\n#error _GLIBCXX_HAS_GTHREADS\n#endif"
-            COMMAND "${CMAKE_CXX_COMPILER}" -x c++ -E -
-            RESULT_VARIABLE STATUS_CXX11_THREADS
-            OUTPUT_QUIET
-            ERROR_QUIET
-        )
-        if (NOT STATUS_CXX11_THREADS EQUAL 0)
-            # https://stackoverflow.com/a/28001271
-            set (CMAKE_CXX_STANDARD_LIBRARIES "-Wl,-Bstatic -lstdc++ -lpthread -Wl,-Bdynamic ${CMAKE_CXX_STANDARD_LIBRARIES}")
-        endif ()
+#        execute_process (
+#            COMMAND "${CMAKE_COMMAND}" -E echo "#include <thread>\n#ifdef _GLIBCXX_HAS_GTHREADS\n#error _GLIBCXX_HAS_GTHREADS\n#endif"
+#            COMMAND "${CMAKE_CXX_COMPILER}" -x c++ -E -
+#            RESULT_VARIABLE STATUS_CXX11_THREADS
+#            OUTPUT_QUIET
+#            ERROR_QUIET
+#        )
+#        if (NOT STATUS_CXX11_THREADS EQUAL 0)
+#            # https://stackoverflow.com/a/28001271
+#            set (CMAKE_CXX_STANDARD_LIBRARIES "-Wl,-Bstatic -lstdc++ -lpthread -Wl,-Bdynamic ${CMAKE_CXX_STANDARD_LIBRARIES}")
+#        endif ()
     endif ()
 endif ()
 
