diff --git a/CMakeLists.txt b/CMakeLists.txt
index ff7d3e9..a7e9f0d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -50,13 +50,13 @@ find_package(KF5I18n ${KF5_VERSION} REQUIRED)
 
 if (NOT WIN32)
     find_package(KF5WidgetsAddons ${KF5_VERSION} REQUIRED)
-    find_package(KF5TextWidgets ${KF5_VERSION} REQUIRED)
+#    find_package(KF5TextWidgets ${KF5_VERSION} REQUIRED)
 
     # Optional KF5 frameworks
-    find_package(KF5Wallet ${KF5_VERSION})
-    set_package_properties(KF5Wallet PROPERTIES DESCRIPTION "Safe desktop-wide storage for passwords" TYPE OPTIONAL
-        PURPOSE "Required to have permanent storage of passwords for scanners"
-    )
+#    find_package(KF5Wallet ${KF5_VERSION})
+#    set_package_properties(KF5Wallet PROPERTIES DESCRIPTION "Safe desktop-wide storage for passwords" TYPE OPTIONAL
+#        PURPOSE "Required to have permanent storage of passwords for scanners"
+#    )
 
     # Check if sane API is available.
     find_package(KSaneCore "${RELEASE_SERVICE_VERSION_MAJOR}.${RELEASE_SERVICE_VERSION_MINOR}" REQUIRED)
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 9383997..5d9178b 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -51,7 +51,7 @@ target_link_libraries(KF5Sane
     PRIVATE
         KF5::I18n
         KF5::WidgetsAddons
-        KF5::TextWidgets
+#        KF5::TextWidgets
         KSane::Core
 )
 
diff --git a/src/widgets/labeledslider.cpp b/src/widgets/labeledslider.cpp
index 9f71f90..486dfb4 100644
--- a/src/widgets/labeledslider.cpp
+++ b/src/widgets/labeledslider.cpp
@@ -10,10 +10,12 @@
 
 #include <QLabel>
 #include <QSlider>
+#include <QSpinBox>
 
 // KDE includes
 
-#include <KPluralHandlingSpinBox>
+//#include <KPluralHandlingSpinBox>
+#include <klocalizedstring.h>
 #include <Option>
 
 namespace KSaneIface
@@ -90,7 +92,7 @@ void LabeledSlider::initSlider(int minValue, int maxValue, int stepValue)
     m_slider->setMaximum(maxValue);
     m_slider->setSingleStep(m_step);
 
-    m_spinb = new KPluralHandlingSpinBox(this);
+    m_spinb = new QSpinBox(this);
     m_spinb->setMinimum(minValue);
     m_spinb->setMaximum(maxValue);
     m_slider->setSingleStep(m_step);
@@ -114,7 +116,7 @@ void LabeledSlider::initSlider(int minValue, int maxValue, int stepValue)
 
 void LabeledSlider::setSuffix(const KLocalizedString &text)
 {
-    m_spinb->setSuffix(text);
+    m_spinb->setSuffix(text.toString());
 }
 
 void LabeledSlider::setValue(const QVariant &val)
diff --git a/src/widgets/labeledslider.h b/src/widgets/labeledslider.h
index fe1bd4c..b9f5460 100644
--- a/src/widgets/labeledslider.h
+++ b/src/widgets/labeledslider.h
@@ -10,7 +10,8 @@
 
 #include "ksaneoptionwidget.h"
 
-class QSlider;
+#include <QSlider>
+#include <QSpinBox>
 
 /**
   *@author Kåre Särs
@@ -78,7 +79,7 @@ Q_SIGNALS:
 private:
     void initSlider(int minValue, int maxValue, int stepValue);
     QSlider   *m_slider;
-    KPluralHandlingSpinBox  *m_spinb;
+    QSpinBox  *m_spinb;
     int        m_step;
 };
 
